app/views/TaxReturnReceivedView.scala.html

```diff
@@ -61,20 +60,12 @@
                     classes = "govuk-!-width-one-quarter",
                     content = HtmlContent(timeStamp)
                 )
-            ),
-            SummaryListRow(
-                key = Key(
-                    classes = "govuk-!-width-three-quarters",
-                    content = Text(messages("taxReturnReceived.summaryrow.4"))
-                ),
-                value = Value(
-                    classes = "govuk-!-width-one-quarter",
-                    content = HtmlContent(bigDecimalCurrency(summaryData.incomeTaxAndNationalInsuranceContributions.toString))
-                )
             )
         )
     ))
 
+    <br>
```

Comment:
I don't think using <br> tags to create spacing between paragraphs is the best way to go about it. Can cause problems for people using screen readers

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
fixed All